### PR TITLE
Vaillant recoVair 260

### PR DESCRIPTION
**Pull request by kmplngj**
I added the config for Vaillant recoVair 260. It is based on the old pull request https://github.com/john30/ebusd-configuration/pull/102. I did the changes I could do. Perhaps someone could have a look at it. I would be happy if I could help adding this.

### CODE REVIEW

**Comment by karlkashofer** [comment]
Hi ! I use that config with my recovair 260, but i think the FlowActual:
`r1,recov,FlowActual,Current Volume Flow,,,,29030e,,,UIN,,m3/h,`
is not working.
Its always between 120 and 124 regardless of the real flow shown on the display. 
It also does not react when i SetVentBoost.
Can you confirm that is working for you ?

**Comment by karlkashofer** [comment]
Hi ! 
I use that config with my recovair 260, but i think the FlowActual:
`r1,recov,FlowActual,Current Volume Flow,,,,29030e,,,UIN,,m3/h,`
is not working correctly. 

The values i get from ebusd are correlated, but not equal to the values on the hardware display:

Stufe 5 => Display: 192, EbusD: 155-157
Stufe 4 => Display: 175, EbusD: 120-122
Stufe 2 => Display: 140, EbusD: 117-120

Do you also see this ?
Do you have any idea what could be the reason ?

**Comment by kmplngj** [comment]
@karlkashofer I think the ebus shows the current flow while the display shows the rough setting, which is always regulated again depending on the measured air humidity.

File: ebusd-2.x.x/de/vaillant/08.recov.csv [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# type (r[1-9];w;u),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment
```
Comment (john30) [inline ebusd-2.x.x/de/vaillant/08.recov.csv:1]: 2.x.x is only kept for ease of accessing older files, so please remove it from the PR
same applies to the EN file in that folder

File: ebusd-2.1.x/en/vaillant/08.recov.csv [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# type (r[1-9];w;u),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment
+#,recov,recoVair 260,,,,,,,,,,,
+# ##### Generell #####,,,,,,,,,,,,,
+*r,,,,,,B513,,,,IGN:2,,,
+*w,,,,,,B509,,,,,,,
+r,recov,VentIntensity,Read Ventilation Intensity,,,,048a03,,,UCH,,,
+r,recov,VentHeatRecovery,Read Heat Recovery,,,,048c03,,,UCH,0=auto;1=an;2=aus,,
```
Comment (john30) [inline ebusd-2.1.x/en/vaillant/08.recov.csv:7]: value items should be english as well

**Review by john30 — CHANGES_REQUESTED** [review]

**Comment by kmplngj** [comment]
@john30 I need your help.

Since one of the last updates (ebusd HA add-on or the latest ebus adapter 3 Firmware Update) most the mqtt messages do no longer work.

Here is a log. Here you could see TempWasteAir, PreferredTemp, TempInletAir, TempOutsideAir, TempOutgoingAir are work the rest does not get messages.

`
> ebusd --foreground --mqtthost=core-mosquitto --mqttport=1883 --mqttuser=addons --mqttpass=xxxx --mqttretain --configpath=/config/misc/ebusd/ebusd-configuration/ebusd-2.1.x/de/ --latency=10 --pollinterval=2 --mqttint=/etc/ebusd/mqtt-hassio.cfg --mqtttopic=ebusd --device=enh:192.168.188.130:9999 --log=all:notice --scanconfig=08
2023-10-02 10:18:50.260 [main notice] ebusd 23.2.p20230716 started with single scan on device: 192.168.188.130:9999, enhanced
2023-10-02 10:18:50.290 [bus notice] device status: resetting
2023-10-02 10:18:50.290 [bus notice] bus started with own address 31/36
2023-10-02 10:18:50.296 [bus notice] signal acquired
2023-10-02 10:18:50.437 [mqtt notice] connection established
2023-10-02 10:18:50.491 [bus notice] device status: reset
2023-10-02 10:19:00.292 [main notice] starting initial scan for 08
2023-10-02 10:19:00.495 [bus notice] new master 03, master count 2
2023-10-02 10:19:00.495 [bus notice] scan 08: ;Vaillant;recoV;0225;0203
2023-10-02 10:19:00.495 [update notice] store 08 ident: done
2023-10-02 10:19:00.495 [update notice] sent scan-read scan.08  QQ=31: Vaillant;recoV;0225;0203
2023-10-02 10:19:00.495 [bus notice] scan 08: ;Vaillant;recoV;0225;0203
2023-10-02 10:19:00.498 [main notice] read common config file vaillant/general.csv
2023-10-02 10:19:00.498 [main notice] read common config file vaillant/scan.csv
2023-10-02 10:19:00.499 [main notice] read common config file vaillant/broadcast.csv
2023-10-02 10:19:00.500 [main notice] read scan config file vaillant/08.recov.csv for ID "recov", SW0225, HW0203
2023-10-02 10:19:00.500 [main notice] found messages: 50 (0 conditional on 0 conditions, 0 poll, 8 update)
2023-10-02 10:19:00.684 [update notice] sent unknown MS cmd: 3108b5090124 / 09003231313734383030
2023-10-02 10:19:00.876 [update notice] sent scan-read scan.08 id QQ=31: 
2023-10-02 10:19:01.063 [update notice] sent scan-read scan.08 id QQ=31: 
2023-10-02 10:19:01.278 [update notice] sent scan-read scan.08 id QQ=31: 21;17;48;0010016040;0001;005334;N6
2023-10-02 10:19:01.278 [bus notice] scan 08: ;21;17;48;0010016040;0001;005334;N6
2023-10-02 10:19:01.278 [main notice] initial scan result: 08;Vaillant;recoV;0225;0203;21;17;48;0010016040;0001;005334;N6
2023-10-02 10:19:06.576 [update notice] sent poll-read recov PreferredTemp QQ=31: 20.50
2023-10-02 10:19:09.214 [update notice] sent poll-read recov TempInletAir QQ=31: 20.12;ok
2023-10-02 10:19:12.211 [update notice] sent poll-read recov TempOutgoingAir QQ=31: 22.06;ok
2023-10-02 10:19:15.188 [update notice] sent poll-read recov TempOutsideAir QQ=31: 19.50;ok
2023-10-02 10:19:18.206 [update notice] sent poll-read recov TempWasteAir QQ=31: 22.06;ok
2023-10-02 10:19:21.183 [update notice] sent poll-read recov PreferredTemp QQ=31: 20.50
2023-10-02 10:19:24.202 [update notice] sent poll-read recov TempInletAir QQ=31: 20.12;ok
2023-10-02 10:19:27.219 [update notice] sent poll-read recov TempOutgoingAir QQ=31: 22.06;ok
2023-10-02 10:19:30.207 [update notice] sent poll-read recov TempOutsideAir QQ=31: 19.50;ok
2023-10-02 10:19:33.186 [update notice] sent poll-read recov TempWasteAir QQ=31: 22.06;ok
2023-10-02 10:19:36.211 [update notice] sent poll-read recov PreferredTemp QQ=31: 20.50
2023-10-02 10:19:39.194 [update notice] sent poll-read recov TempInletAir QQ=31: 20.12;ok
2023-10-02 10:19:42.211 [update notice] sent poll-read recov TempOutgoingAir QQ=31: 22.06;ok
2023-10-02 10:19:45.209 [update notice] sent poll-read recov TempOutsideAir QQ=31: 19.50;ok
2023-10-02 10:19:48.182 [update notice] sent poll-read recov TempWasteAir QQ=31: 22.06;ok
2023-10-02 10:19:51.193 [update notice] sent poll-read recov PreferredTemp QQ=31: 20.50
2023-10-02 10:19:54.221 [update notice] sent poll-read recov TempInletAir QQ=31: 20.12;ok
2023-10-02 10:19:57.195 [update notice] sent poll-read recov TempOutgoingAir QQ=31: 22.06;ok
2023-10-02 10:20:00.216 [update notice] sent poll-read recov TempOutsideAir QQ=31: 19.50;ok
2023-10-02 10:20:00.543 [update notice] sent write recov SetVentDay QQ=31: 
2023-10-02 10:20:00.543 [mqtt notice] write recov SetVentDay: 
2023-10-02 10:20:00.543 [mqtt error] decode recov SetVentDay: empty
2023-10-02 10:20:01.544 [mqtt error] decode recov SetVentDay: empty
2023-10-02 10:20:03.214 [update notice] sent poll-read recov TempWasteAir QQ=31: 22.06;ok
2023-10-02 10:20:06.417 [update notice] sent poll-read recov PreferredTemp QQ=31: 20.50
2023-10-02 10:20:09.187 [update notice] sent poll-read recov TempInletAir QQ=31: 20.19;ok
2023-10-02 10:20:12.203 [update notice] sent poll-read recov TempOutgoingAir QQ=31: 22.06;ok
2023-10-02 10:20:15.174 [update notice] sent poll-read recov TempOutsideAir QQ=31: 19.38;ok
2023-10-02 10:20:18.196 [update notice] sent poll-read recov TempWasteAir QQ=31: 22.19;ok
2023-10-02 10:20:21.210 [update notice] sent poll-read recov PreferredTemp QQ=31: 20.50
2023-10-02 10:20:24.191 [update notice] sent poll-read recov TempInletAir QQ=31: 20.19;ok
2023-10-02 10:20:27.208 [update notice] sent poll-read recov TempOutgoingAir QQ=31: 22.06;ok
2023-10-02 10:20:30.190 [update notice] sent poll-read recov TempOutsideAir QQ=31: 19.38;ok
2023-10-02 10:20:33.251 [update notice] sent poll-read recov TempWasteAir QQ=31: 22.19;ok
2023-10-02 10:20:36.191 [update notice] sent poll-read recov PreferredTemp QQ=31: 20.50
2023-10-02 10:20:39.215 [update notice] sent poll-read recov TempInletAir QQ=31: 20.19;ok
2023-10-02 10:20:42.198 [update notice] sent poll-read recov TempOutgoingAir QQ=31: 22.06;ok
2023-10-02 10:20:45.219 [update notice] sent poll-read recov TempOutsideAir QQ=31: 19.38;ok
2023-10-02 10:20:48.199 [update notice] sent poll-read recov TempWasteAir QQ=31: 22.19;ok
2023-10-02 10:20:51.210 [update notice] sent poll-read recov PreferredTemp QQ=31: 20.50
2023-10-02 10:20:51.474 [main notice] update check: revision 23.2 available
2023-10-02 10:20:54.188 [update notice] sent poll-read recov TempInletAir QQ=31: 20.19;ok
2023-10-02 10:20:57.233 [update notice] sent poll-read recov TempOutgoingAir QQ=31: 22.06;ok
2023-10-02 10:21:00.209 [update notice] sent poll-read recov TempOutsideAir QQ=31: 19.38;ok
2023-10-02 10:21:03.183 [update notice] sent poll-read recov TempWasteAir QQ=31: 22.19;ok
2023-10-02 10:21:06.218 [update notice] sent poll-read recov PreferredTemp QQ=31: 20.50
2023-10-02 10:21:09.193 [update notice] sent poll-read recov TempInletAir QQ=31: 20.19;ok
2023-10-02 10:21:12.211 [update notice] sent poll-read recov TempOutgoingAir QQ=31: 22.06;ok
2023-10-02 10:21:15.188 [update notice] sent poll-read recov TempOutsideAir QQ=31: 19.38;ok
2023-10-02 10:21:18.210 [update notice] sent poll-read recov TempWasteAir QQ=31: 22.19;ok
2023-10-02 10:21:21.183 [update notice] sent poll-read recov PreferredTemp QQ=31: 20.50
2023-10-02 10:21:24.205 [update notice] sent poll-read recov TempInletAir QQ=31: 20.19;ok
2023-10-02 10:21:27.184 [update notice] sent poll-read recov TempOutgoingAir QQ=31: 22.06;ok`

**Review by john30 — CHANGES_REQUESTED** [review]
please remove irrelevant files (.DS_Store) in several locations

**Comment by john30** [comment]
can you share your scan result please @kmplngj ?

**Comment by kmplngj** [comment]
@john30 do you have a link to a guide how to do the scan? It is so long ago that I did the setup.

**Comment by john30** [comment]
just issue an "info" command and the result shows the scan infos as well.

**Comment by kmplngj** [comment]
info
version: ebusd 23.2.p20230716
update check: version 24.1 available, vaillant/08.recov.csv: newer version available
device: 192.168.188.130:9999, enhanced
signal: acquired
symbol rate: 37
max symbol rate: 114
min arbitration micros: 3
max arbitration micros: 12
min symbol latency: 8
max symbol latency: 38
scan: finished
reconnects: 0
masters: 2
messages: 44
conditional: 0
poll: 12
update: 8
address 03: master #11
address 08: slave #11, scanned "MF=Vaillant;ID=recoV;SW=0225;HW=0203", loaded "vaillant/08.recov.csv"
address 31: master #8, ebusd
address 36: slave #8, ebusd

scan result
08;Vaillant;recoV;0225;0203;21;17;48;0010016040;0001;005334;N6

**Comment by john30** [comment]
please check the [next](https://ebus.github.io/next/) generated csvs with ebusd 24.1 (or current source) as described there or by using a local clone of https://github.com/eBUS/ebus.github.io and using the "next" folder in there

**Pull request merged**